compute/containers/how-to/add-a-custom-domain-to-a-container.mdx

```diff
@@ -20,7 +20,16 @@ Each container comes with an automatically generated domain on which it is avail
 
 </Message>
 
-1. Add a CNAME pointing to your container default URL from the [domains dashboard](/network/dns-cloud/how-to/manage-dns-records/).
+1. Add a CNAME pointing to your container default URL in your DNS Record (for example [Scaleway's domains](/network/dns-cloud/how-to/manage-dns-records/)).
+
+<Message type="note">
+- All domains are supported.
+- CNAME flattening* is supported.
+- If your DNS provider does not support CNAME flattening, you need to create an ALIAS instead of a CNAME.
```

Comment:
```suggestion
- If your DNS provider does not support CNAME flattening, you need to create an ALIAS on your root domain (named `@` or with an empty name) instead of a CNAME.
```